extend/element-types.md

```diff
@@ -857,3 +857,24 @@ public function setEagerLoadedElements(string $handle, array $elements)
     }
 }
 ```
+
+## Saving Field Content Deltas
+
+Element forms can be configured to submit only the field values that actually changed on the page. This is a prerequisite to [field delta saves](/extend/field-types.md#supporting-delta-saves).
+
+If your element provides its own edit form template, here’s how you can configure it to submit delta field content:
+
+1. Enable delta input name registration at the top of your template.
+2. Add `registerDeltas: true` wherever you’ve used `_includes/fields.html` or `_includes/field.html`.
+
+```twig{1,6}
+{% do view.setIsDeltaRegistrationActive(true) %}
+
```

Comment:
Maybe put a `...` line here surrounded by blank lines, so it’s more clear that these wouldn’t go right next to each other. (Don’t forget to change `{1,6}` to `{1,8}` in the process.)